src/main/drivers/timer.c

```diff
@@ -249,7 +251,8 @@ void timerConfigure(const timerHardware_t *timerHardwarePtr, uint16_t period, ui
 {
     configTimeBase(timerHardwarePtr->tim, period, mhz);
     TIM_Cmd(timerHardwarePtr->tim, ENABLE);
-
+if (timerHardwarePtr->tim == TIM7)
+return;
```

Comment:
Ah, yeah, to be precise, it interrupts, and hangs.

These two lines were added to see the source of hanging.
```
    uint8_t irq = timerInputIrq(timerHardwarePtr->tim);
    timerNVICConfigure(irq);
```